thrift/src/main/thrift/contentatom.thrift

```diff
@@ -20,7 +21,8 @@ enum AtomType {
   CTA = 4,
   INTERACTIVE = 5,
   REVIEW = 6,
-  RECIPE = 7
+  RECIPE = 7, 
+  STORYQUESTIONS = 8
```

Comment:
I'm a bit surprised that a question isn't an atom but instead we have multiple questions in a single atom. Is it the case that you would always want all the questions asked to be bundled up together?